Makefile

```diff
@@ -204,9 +204,14 @@ remote-deps: mod-download
 		cp bin/bpf/bpf-apache/bin/* filesystem/usr/lib/calico/bpf/; \
 		chmod -R +w filesystem/etc/calico/confd/ config/ filesystem/usr/lib/calico/bpf/'
 
+TARGET_PLATFORM=${ARCH}
 # We need CGO when compiling in Felix for BPF support.  However, the cross-compile doesn't support CGO yet.
 ifeq ($(ARCH), amd64)
 CGO_ENABLED=1
+else ifeq ($(ARCH), arm64)
+# eBPF cross-complie for arm64/aarch64.
+CGO_ENABLED=1
+TARGET_PLATFORM=arm64/v8
```

Comment:
Might be worth a comment explaining why TARGET_PLATFORM is different for arm64 - what does the /v8 do and why do we not need it for other architectures?

(Sorry if I'm a bit dense)

Reply:
Well since I have a strong history of breaking things :D and only tested this againts arm64v8 and with the arrival of arm64/9 I wanted to discourage that build until it could be tested.

I believe `arm64 = latest`